EKF/control.cpp

```diff
@@ -202,14 +202,14 @@ void Ekf::controlExternalVisionFusion()
 				if (_params.fusion_mode & MASK_USE_EVPOS && !_control_status.flags.ev_pos) {
 					_control_status.flags.ev_pos = true;
 					resetPosition();
-					ECL_INFO_TIMESTAMPED("commencing external vision position fusion");
+					ECL_INFO_TIMESTAMPED("commencing vision position fusion");
```

Comment:
You could even shorten that to "Vision pos fusion started" or "starting vision pos fusion"

Reply:
that is even better.